### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by fncischen**
@ryanboris

### CODE REVIEW

File: src/components/TodoComponents/TodoForm.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    return (
+        <form onSubmit={props.addTask}>
+        
+        <input type ="text"
+        value = {props.task}
+        name = "task"
+        placeholder = "Task"
+        onChange = {props.handleChanges}
+        ></input>
```
Comment (secantsquared) [inline src/components/TodoComponents/TodoForm.js:13]: Well handled.

File: src/components/TodoComponents/TodoList.js [hunk 2]
```diff
@@ -1,2 +1,98 @@
...
+            task: this.state.task,
+            id: Date.now(),
+            completed: false
+        }
+
+        console.log(newTask);
+        // call setState to tell the computer directly 
+        this.setState({
+            tasks: [... this.state.tasks, newTask],
+        });
```
Comment (secantsquared) [inline src/components/TodoComponents/TodoList.js:47]: Nice fix..  you definitely are getting it , well done.

File: src/components/TodoComponents/TodoList.js [hunk 3]
```diff
@@ -1,2 +1,98 @@
...
+
+
+    }
+
+    // track changes on the form :)
+    handleChanges = e => {
+        console.log(e.target.value);
+        //     this.setState({ name: e.target.value })
+        this.setState({
+          [e.target.name]: e.target.value
```
Comment (secantsquared) [inline src/components/TodoComponents/TodoList.js:64]: Good use of the name property to differentiate between different inputs.

**Review by secantsquared — APPROVED** [review]
Nice work, you are starting to get into the groove.

**Comment by secantsquared** [comment]
Day 1 MVP met continue Day 2 on new branch.. awesome stuff

File: src/components/TodoComponents/TodoList.js (hunk 3 above)
Comment (fncischen) [reply src/components/TodoComponents/TodoList.js:64]: thanks!

**Pull request merged**